docs/DataWorkflows/FetchingYourData.md

```diff
@@ -143,8 +141,7 @@ sub-colornest002
 sub-colornest195
 ```
 
-This directory contains all the outputs from xcp_d for each subject. Now, let's see if we can find the
-actual file we want to get for each subject. We can see all the files for a single subject with `find`
+This directory contains all the outputs from xcp_d for each subject. Now, let's see if we can find the actual file we want to get for each subject. We can see all the files for a single subject with `find`
```

Comment:
I would add a subtitle in the text above, in this way:

This directory contains all the outputs from xcp_d for each subject. 

### Figuring out what data is available (single participant)
Now, let's see if we can find the actual file we want to get for each subject. We can see all the files for a single subject with `find`: